runtime/plugin/matchparen.vim

```diff
@@ -18,14 +18,27 @@ if !exists("g:matchparen_insert_timeout")
   let g:matchparen_insert_timeout = 60
 endif
 
+if !exists('g:matchparen_show_delay')
+  let g:matchparen_show_delay = 100
+endif
+if !exists('g:matchparen_hide_delay')
```

Comment:
We should document these configuration variables.
File runtime/doc/pi_paren.txt looks like the right place.